mitodl/mitxonline
# Redesign footer

# What are the relevant tickets?




https://github.com/mitodl/hq/issues/2251

# Description (What does it do?)

Adds the feature-flagged new footer to all pages based on design

# Screenshots (if appropriate):

 Desktop screenshots
![image](https://github.com/mitodl/mitxonline/assets/3138890/5d872e1a-11c1-4832-9d1c-d9f73f20c4d4)

Mobile width screenshots
![image](https://github.com/mitodl/mitxonline/assets/3138890/e5cbfc0f-9214-402b-acf3-76e4ca01d894)


# How can this be tested?

Make sure `POSTHOG_API_HOST` and `POSTHOG_API_TOKEN` are added to .env. `mitxonline-new-footer` flag is turned on dev, so you should be able to see the new footer with this branch

# Additional Context



[figma](https://www.figma.com/file/Gs4zIhOFv5gVvafawwl6PF/MITx-Online?type=design&node-id=558-2708&mode=design&t=b0qqwwCF9geNHFnm-0)
